ibmdbpy/ae/groupedapply.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, df, index, output_signature, output_table=None, fun_ref=None,
         self.db = df._idadb
         self.fun = fun_name
```

Comment:
This should be  `self.fun = fun_ref`. 
Can you add a testcase for NZFunGroupedApply with the fun_ref parameter set?